casbin/enforcer.cpp

```diff
@@ -69,6 +68,8 @@ bool Enforcer :: enforce(string matcher, Scope scope) {
         }
     }
 
+    this->func_map.ApplyFunctionMap();
```

Comment:
@bigcat26 I didn't remember that we have `ApplyFunctionMap()` in Go or Java Casbin, can you copy the implementation from other languages?

Reply:
OK, maybe I fix it in another way.

Reply:
There isn't batch action for matching_function in other version of Casbin, maybe this can be an example?